.github/workflows/test.yml

```diff
@@ -78,11 +78,21 @@ jobs:
 
     - name: Run acceptance test
       run: make testacc
+  coverage:
```

Comment:
I think this is from your other PR. Is that intended here?

Reply:
not completely. I found the issue of why some of the jobs are failing sometimes (it is due to their concurrent execution) and I wanted to fix it through this PR but I'm gonna split it into a different PR so I'm gonna remove this from his PR soon